src/js/utils/styles.js

```diff
@@ -700,17 +700,20 @@ const ROUND_MAP = {
 
 export const roundStyle = (data, responsive, theme) => {
   const breakpoint = getBreakpointStyle(theme, theme.box.responsiveBreakpoint);
+  // fallback to edgeSize for backwards compatibility
+  const radius = theme.global.radius ? 'radius' : 'edgeSize';
+
   const styles = [];
   if (typeof data === 'object') {
     const size =
       ROUND_MAP[data.size] ||
-      theme.global.edgeSize[data.size || 'medium'] ||
+      theme.global[radius][data.size || 'medium'] ||
       data.size;
     const responsiveSize =
       responsive &&
       breakpoint &&
-      breakpoint.edgeSize[data.size] &&
-      (breakpoint.edgeSize[data.size] || data.size);
+      breakpoint[radius]?.[data.size] &&
```

Comment:
What happens if `theme.global.radius` is defined but `theme.global.breakpoints[size].radius` isn't defined? Should we fall back to looking at `theme.global.breakpoints[size].edgeSize`

Reply:
If `responsiveSize` comes back as undefined, then it won't apply the "responsniveStyling" and just will apply from `theme.global.radius` (this would be in alignment with how currently edgeSize and responsive edge size behaves).

Let me know if that makes sense.

Reply:
Ah okay that makes sense thanks!